contrib/FreeBSD/README

```diff
@@ -0,0 +1,46 @@
+This folder contains sopel and sopel-default.cfg designed to be distributed by third parties such as FreeBSD.
+
+`sopel-default.cfg` is a default configuration file for sopel. The -default prefix is mandatory and will be used as a 'profile'. The profiles may be changed with the variable `sopel_profiles` in `/etc/rc.conf`. For example:
+
+```sh
+sysrc sopel_profiles="profile1 profile2 profile3"
+```
+
+The configuration files should be `${sopel_confdir}/${sopel_prefix}profile1.cfg`, `${sopel_confdir}/${sopel_prefix}profile2.cfg` and `${sopel_confdir}/${sopel_prefix}profile3.cfg`. The variable `sopel_confdir` is the default configuration directory (by default `/usr/local/etc`). `sopel_prefix`is the prefix for all files (by default `sopel-`).
```

Comment:
Is editing these a routine enough use case to warrant documenting it this way? I'd prefer to avoid making people reading docs parse variable syntax. Also, GitHub handles long lines *horribly*, I'd suggest sticking to 80-120 chars.
Phrasing:
```suggestion
The configuration files should be `${sopel_confdir}/${sopel_prefix}profile1.cfg`, `${sopel_confdir}/${sopel_prefix}profile2.cfg` and `${sopel_confdir}/${sopel_prefix}profile3.cfg`. The variable `sopel_confdir` is the configuration directory (by default `/usr/local/etc`). `sopel_prefix`is the prefix for all files (by default `sopel-`).
```

Reply:
I could change the content of the README to make it easier to understand without the need to parse the variables. I will keep between 80-120 chars